custom/icds_reports/migrations/sql_templates/database_functions/insert_into_child_health_monthly.sql

```diff
@@ -225,6 +225,8 @@ BEGIN
     'WHERE chm_monthly.case_id = agg.case_id AND chm_monthly.valid_in_month = 1 AND agg.month = ' || quote_literal(_start_date);
 
     EXECUTE 'UPDATE ' || quote_ident(_tablename) || ' chm_monthly SET ' ||
+      'recorded_height = CASE WHEN (date_trunc(' || quote_literal('MONTH') || ', agg.height_child_last_recorded) = ' || quote_literal(_start_date) || ') THEN agg.height_child ELSE NULL END, ' ||
```

Comment:
Confusingly `recorded_height` actually means "height of the child if it was recorded in this month"